firefox-ios/Client/Frontend/Fakespot/Views/FakespotMessageCardView.swift

```diff
@@ -153,28 +153,25 @@ final class FakespotMessageCardView: UIView, ThemeApplicable, Notifiable {
         label.setContentCompressionResistancePriority(.required, for: .vertical)
     }
 
-    private lazy var linkButton: LegacyResizableButton = .build { button in
+    private lazy var linkButton: LinkButton = .build { button in
         button.titleLabel?.font = DefaultDynamicFontHelper.preferredFont(
             withTextStyle: .caption1,
             size: UX.linkFontSize)
-        button.buttonEdgeSpacing = 0
         button.contentHorizontalAlignment = .leading
         button.addTarget(self, action: #selector(self.linkAction), for: .touchUpInside)
-        button.titleLabel?.adjustsFontForContentSizeCategory = true
     }
 
-    private lazy var primaryButton: LegacyResizableButton = .build { button in
+    private lazy var primaryButton: PrimaryRoundedButton = .build { button in
         button.titleLabel?.font = DefaultDynamicFontHelper.preferredBoldFont(
```

Comment:
The font that is already configured in the `PrimaryRoundedButton` should not be overwritten.

Reply:
Same as for the `LinkButton`: Could you please remove the setting of the font as well? As we are using a standard component we should use the font of specified by the component.